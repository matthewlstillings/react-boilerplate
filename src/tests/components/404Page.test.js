import React from 'react';
import {shallow} from 'enzyme';
import NotFoundPage from '../../components/404page';

test('should render 404 Error Page', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});