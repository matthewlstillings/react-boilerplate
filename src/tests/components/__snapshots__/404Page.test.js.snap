// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 Error Page 1`] = `
<div>
  <p>
    404 Error
  </p>
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
